jobs/build-arch.Jenkinsfile

```diff
@@ -446,38 +385,8 @@ lock(resource: "build-${params.STREAM}-${basearch}") {
                 }
             }
 
-            stage('Test Live ISO') {
-                // compress the metal and metal4k images now so we're testing
-                // installs with the image format we ship
-                shwrap("""
-                cosa compress --compressor xz --artifact metal --artifact metal4k
-                """)
-                try {
-                    parallelruns = [:]
-                    parallelruns['metal'] = {
-                        shwrap("cosa kola testiso -S --output-dir tmp/kola-testiso-metal")
-                    }
-                    // metal4k doesn't work on s390x right now: https://github.com/coreos/fedora-coreos-tracker/issues/1261
-                    if (basearch != "s390x") {
-                        parallelruns['metal4k'] = {
-                            shwrap("cosa kola testiso -SP --qemu-native-4k --qemu-multipath --output-dir tmp/kola-testiso-metal4k")
-                        }
-                    }
-                    // process this batch
-                    parallel parallelruns
-                } catch (Throwable e) {
-                    throw e
-                } finally {
-                    shwrap("""
-                    cosa shell -- tar -c --xz tmp/kola-testiso-metal/ > kola-testiso-metal.tar.xz
-                    cosa shell -- tar -c --xz tmp/kola-testiso-metal4k/ > kola-testiso-metal4k.tar.xz || :
-                    """)
-                    archiveArtifacts allowEmptyArchive: true, artifacts: 'kola-testiso*.tar.xz'
-                }
-            }
-
-            // reset for the next batch of independent tasks
-            parallelruns = [:]
+            // Run Kola TestISO tests for metal artifacts
+            kolaTestIso(cosaDir: env.WORKSPACE, build: newBuildID, arch: basearch)
```

Comment:
I don't think `build` does anything for `kolaTestIso`.

Reply:
Correct, fixed.